web/src/features/fba/components/AdvisoryMetadata.tsx

```diff
@@ -0,0 +1,45 @@
+import { Autocomplete, Box, Grid, TextField, Typography } from '@mui/material'
+import React from 'react'
+import { DateTime } from 'luxon'
+import { RunType } from 'features/fba/pages/FireBehaviourAdvisoryPage'
+import { isNull } from 'lodash'
+
+export interface AdvisoryMetadataProps {
+  testId?: string
```

Comment:
setIssueDate will be used in an upcoming ticket (I was a little pre-emptive in adding it now, but I figured it sort of makes sense to add it now because we're registering the issueDate prop in React.State)

testId is an optional prop that we typically add to all component props. If we ever go back and write unit tests for ASA (which we should...) then testId should come in useful.